analyzer.go

```diff
@@ -86,14 +86,28 @@ func (a *Analyzer) skipCodeGen(file *ast.File) ([]*ast.CommentGroup, []*ast.Comm
 	if len(comments) > 0 {
 		list = comments[0].List
 	}
-	if len(comments) > 0 && strings.Contains(comments[0].Text(), "DO NOT EDIT") {
+	if len(comments) > 0 && (strings.Contains(comments[0].Text(), "DO NOT EDIT") || strings.Contains(comments[0].Text(), "go:build")) {
 		comments = comments[1:]
 		list = comments[0].List
 		if len(list) > 0 && strings.HasSuffix(list[0].Text, "//line:") {
 			list = list[1:]
 		}
 	}
 
+	for len(list) > 0 {
+		if strings.Contains(list[0].Text, "go:build") {
```

Comment:
same thing here.

Reply:
Fixed.

Reply:
not fixed. https://github.com/denis-tingaikin/go-header/pull/51#discussion_r2094610321

Reply:
added https://github.com/denis-tingaikin/go-header/pull/51/commits/c3631e75f81e202d4bc3a5834f7d9fe39fd32826